packages/hosted-field-integration/src/hosted-field-integration.tsx

```diff
@@ -49,60 +54,58 @@ export class HostedFieldPaymentMethodComponent extends Component<HostedFieldPaym
         } catch (error) {
             onUnhandledError(error);
         }
-    }
+    };
+
```

Comment:
This looks good, but can we try and move this to a package and use the payment-integration-api?